### DISCUSSION TITLE
Using Ambe Vocoder with HR-C6000

### DISCUSSION THREAD

**Discussion by JozefRaedo** [community/community#18921] [Programming Help]
Hi every one,
Im Jozef Raedo, student in telecommunication engineering and Im beginner in this filed, A few weeks ago I was working in using HR-C6000 DMR chip with AT1846S transceiver , but I faced difficulties but I solved it, now I want to implement Ambe Vocoder in STM32F host controller and I need to benefit from the source of MD380, so can you help me please?
I see project in github which has the following heading " Software AMBE+2 2450x1150 vocoder library using md380 firmware" but I need to help me how to use it with my project please.

Jozef

**Comment by alwinhb** [comment]
Hi @JozefRaedo "R-C6000 DMR chip with AT1846S transceiver " Can you help me with this? Were you able to send and receive DMR  Voice/Data?

**Comment by ebndev** [comment]
Hey @JozefRaedo,

Thanks for posting in the Community!

I saw in the code for that repo was linked to another repo [md380tools](https://github.com/travisgoodspeed/md380tools) and their README has more installation instructions and resources. Maybe you can find how to implement the library in your project there.
